tests/flows/test_flow.py

```diff
@@ -205,12 +184,10 @@ def side_effect(self):
             server_xml = server_xml.replace('  ', '').replace('\t', '').strip().replace('\n\n', '\n').replace('&quot;', '"')
             server_xml = re.sub(r'^$', '', server_xml)
 
-
         self.assertEqual(server_xml, local_xml)
 
         self.assertEqual(new_flow, flow)
         self.assertIsNot(new_flow, flow)
-        new_flow.model.fit(X, y)
```

Comment:
I moved it below the asserts, I guess because I felt that it was a better order to have all the string asserts in the same place? I don't have a strong opinion, though.